### ISSUE TITLE
Incorrect x-axis genomic coordinates in --zoom mode 

### ISSUE THREAD

**Issue by ManavalanG** [ryanlayer/samplot#163]
When used with `--zoom` option, samplot's genome coordinates in the x-axis are incorrect. The plot itself appears to be correct and the only problem appears to be with the x-axis coordinates not lining up. 

This is demonstrated here using an example deletion call `chr1:58343207-58343542` using test dataset from this repo `"test/data/hg19_chr1_58343117_58343622_deletion.bam"`. Note that in the zoom mode, start and stop coordinates do not line up with SV call itself, unlike in normal samplot mode (ie. without `--zoom`) and IGV.

**Normal mode:**
![image](https://user-images.githubusercontent.com/12650939/187766960-9e769ef6-2dd4-45fb-a96d-65979abb991c.png)

**Zoom mode:**
![image](https://user-images.githubusercontent.com/12650939/187766925-290b7630-771a-469b-8e98-84b8804e644d.png)

**IGV view:**
![image](https://user-images.githubusercontent.com/12650939/187766707-faf52cdf-6a0d-4183-9499-dc2284b2c3de.png)

**Script:**

```sh
#!/usr/bin/env bash

BAM="test/data/hg19_chr1_58343117_58343622_deletion.bam"    # from samplot repo
CHROM=chr1
START=$(( 58343117+90 ))
END=$(( 58343622-80 ))

ZOOM=" --zoom 100"
OUTFILE="data/${START}-${END}_zoom.png"
# ZOOM=" "
# OUTFILE="data/${START}-${END}.png"

samplot plot \
    -c $CHROM -s $START -e $END \
    --title "CallRegion:${START}-${END}" \
    -b $BAM \
    -o $OUTFILE \
    $ZOOM \
    -t DEL 
```

I tested this using samplot versions `1.1.0` and `1.3.0`, but this bug has been [present since `--zoom` feature was introduced](layerlab.org/2019/06/04/Samplot-improvement.html).

**Comment by jlac** [comment]
Any plans for a fix for this? This is a major issue, limiting the utility of this tool.

Thank you!

Justin

**Comment by mchowdh200** [comment]
yes, we are looking at this along with a couple of other issues and hope to have a new release soon.

**Comment by jsliu1912** [comment]
Hello,

Thank you for this great tool.

I would like to visualize all SVs in a 20Mb window through samplot plot without setting --zoom option. However, I am facing the similar issue where the genomic coordinate doesn’t line up. I am using the samplot version 1.3.0. Is there any way to address this issue? Please see the code and resulting figure below:

source /home/ho/ho_proana/ho_faapf32/miniconda3/etc/profile.d/conda.sh
conda activate samplot

cd /beegfs/work/ho_jason/shared/data/sv-calling/survivor
samplot plot \
    -n 20210113.B-PMS411E_EXACAP 20210113.B-PMS547N_EXACAP \
    -b /beegfs/work/ho_jason/shared/data/mapped_v2.3/20210113.B-PMS411E_EXACAP/gatk/20210113.B-PMS411E_EXACAP.rg.mdup.realign.bam \
       /beegfs/work/ho_jason/shared/data/mapped_v2.3/20210113.B-PMS547N_EXACAP/gatk/20210113.B-PMS547N_EXACAP.rg.mdup.realign.bam \
    -o /beegfs/work/ho_jason/shared/data/sv-calling/INV_EXA.png \
    -c OsphV2_Chr_2  \
    -s 327000000 \
    -e 346000000 \
    -t INV \
    -d 100 \
    --xaxis_label_fontsize 4 \
    -A /beegfs/work/ho_jason/shared/data/sph_ref_genome/Osph-v2.3.fa.gaps.bed.gz

<img width="976" alt="Screenshot 2024-01-29 at 3 50 23 PM" src="https://github.com/ryanlayer/samplot/assets/145424086/df308ec5-4f5f-42b4-b2ca-731b278c5cc7">

(Another minor issue is that the resulting plot still shows all SVs even though I specified showing inversion only.)

Plus, I was wondering is it possible to plot all SVs together in a single plot at a specified region using samplot vcf?

Many thanks,
Jason
